Sources/Services/AWSS3/Package.swift.txt

```diff
@@ -23,6 +23,10 @@ let package = Package(
             id: "aws-sdk-swift.AWSSDKHTTPAuth",
             exact: "0.0.1"
         ),
+        .package(
+            id: "aws-sdk-swift.AWSSDKIdentity",
+            exact: "0.0.1"
+        ),
```

Comment:
Here & below, the generated AWSS3 service client is shown.  There are no changes in any other AWS service client.